lib/support/live_select_web/live/showcase_live.ex

```diff
@@ -134,6 +141,19 @@ defmodule LiveSelectWeb.ShowcaseLive do
           (settings.mode != :single && option == :allow_clear)
       end)
       |> Keyword.new()
+      |> then(&maybe_set_classes_for_multiselect/1)
+    end
+
+    defp maybe_set_classes_for_multiselect(opts) do
+      if LiveSelectWeb.ShowcaseLive.quick_tags?(opts[:mode]) do
+        Keyword.put(
+          opts,
+          :selected_option_class,
+          "cursor-pointer font-bold hover:bg-gray-400 rounded"
+        )
+      else
+        opts
+      end
```

Comment:
Might want to consider adding some new default styles or style options for quick_tags. I added these override styles to the showcase app as the default selected_option style dont work that well with quick_tags.

Reply:
Gotcha. I agree the showcase app should show different styles for the `quick_tags`. However, the way it's implemented right now is a bit too convoluted and has the major drawback that the user can't override the "selected_option_class".

I suggest we do something different, see comment below in the `Settings` module.

(With this new implementation all the above changes can go away)